python/az/aro/azext_aro/custom.py

```diff
@@ -252,11 +258,14 @@ def __init__(self, disk_encryption_id, master_subnet_id, worker_subnet_id):
         if len(errors) > 0:
             logger.error("Issues found blocking cluster creation.\n")
```

Comment:
Given that `az aro create` will assign the needed roles to both the RP SP and the client SP if they are missing them, should information about missing role assignments be printed separately from the other validation failures? That way the information is still there in case customers want to create the role assignments themselves, but it's clear that cluster creation won't be blocked, which is what the output currently indicates.